SDK/AppCenterDistribute/Microsoft.AppCenter.Distribute.Shared/UpdateTrack.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.AppCenter.Distribute
+{
+    public enum UpdateTrack
+    {
+        /// <summary>
+        /// Releases from the public group that don't require authentication.
+        /// </summary>
+        Public = 1,
```

Comment:
Any particular reason we use 1 and 2 here instead of 0 and 1?

Reply:
Yes it's to be future proof if we want to combine them as flags in a future version (for example public | private would mean we would login and fall back to public if login failed).

Reply:
Ah so it's really more like 1 << 1 and 1<< 2 if we were using C terms

Reply:
@jwallra `1 << 0` and `1 << 1` 😉